genai-function-calling/openai-agents/main.py

```diff
@@ -68,5 +68,30 @@ async def main():
     print(result.final_output)
 
 
+async def main():
+    parser = argparse.ArgumentParser(
+        prog="ElasticVersionFetcher",
```

Comment:
nit: maybe match the service name and name in package.json etc. 

```suggestion
        prog="genai-function-calling",
```